docs/plugins-dev-intro.md

```diff
@@ -0,0 +1,40 @@
+# STYX
+## Introduction to developing plugins
+
+This is a quick guide to developing plugins for the Styx platform. It is written with the assumption that you 
+are already familiar with running Styx. If not, please consult the [Quick-start guide](quick-start.md).
+
+We have not yet released our modules online as maven dependencies, but this is coming soon!
+For now, please look in the `lib/` folder inside the ([Styx release](https://github.com/HotelsDotCom/styx/releases)).   
```

Comment:
An alternative, perhaps preferable way is to build the project locally, and using `mvn install` lifecycle phase to install the jars locally. Then the plugin project can just add a dependency in the pom file as they would normally do. 
This is BTW what `make release-no-tests` does.

Reply:
Agreed.